models/repo.go

```diff
@@ -225,13 +225,21 @@ func getReviewers(ctx context.Context, repo *repo_model.Repository, doerID, post
 		// This a private repository:
 		// Anyone who can read the repository is a requestable reviewer
 		if err := e.
-			SQL("SELECT * FROM `user` WHERE id in (SELECT user_id FROM `access` WHERE repo_id = ? AND mode >= ? AND user_id NOT IN ( ?, ?)) ORDER BY name",
+			SQL("SELECT * FROM `user` WHERE id in ("+
```

Comment:
I think we should use xorm builder ...

Reply:
... ok I'll submit a refactor pull

Reply:
thanks! just be careful not to introduce the same bug when the assignee candidate query was refactored: https://github.com/go-gitea/gitea/pull/18994

Reply:
-> #19033